examples/tv-casting-app/android/App/app/src/main/jni/com/matter/casting/support/CommissionerDeclaration.java

```diff
@@ -22,27 +22,30 @@
 public class CommissionerDeclaration {
   static final String TAG = CommissionerDeclaration.class.getSimpleName();
 
-  /** The allowed values for the ErrorCode field are the following */
+  /**
+   * The allowed values for the ErrorCode field are the following. Indicates errors incurred during
+   * commissioning.
+   */
   public enum CdError {
-    kNoError(0),
-    kCommissionableDiscoveryFailed(1),
-    kPaseConnectionFailed(2),
-    kPaseAuthFailed(3),
-    kDacValidationFailed(4),
-    kAlreadyOnFabric(5),
-    kOperationalDiscoveryFailed(6),
-    kCaseConnectionFailed(7),
-    kCaseAuthFailed(8),
-    kConfigurationFailed(9),
-    kBindingConfigurationFailed(10),
-    kCommissionerPasscodeNotSupported(11),
-    kInvalidIdentificationDeclarationParams(12),
-    kAppInstallConsentPending(13),
-    kAppInstalling(14),
-    kAppInstallFailed(15),
-    kAppInstalledRetryNeeded(16),
-    kCommissionerPasscodeDisabled(17),
-    kUnexpectedCommissionerPasscodeReady(18);
+    noError(0),
+    commissionableDiscoveryFailed(1),
+    paseConnectionFailed(2),
+    paseAuthFailed(3),
+    dacValidationFailed(4),
+    alreadyOnFabric(5),
+    operationalDiscoveryFailed(6),
+    caseConnectionFailed(7),
+    caseAuthFailed(8),
+    configurationFailed(9),
+    bindingConfigurationFailed(10),
+    commissionerPasscodeNotSupported(11),
+    invalidIdentificationDeclarationParams(12),
+    appInstallConsentPending(13),
+    appInstalling(14),
+    appInstallFailed(15),
+    appInstalledRetryNeeded(16),
+    commissionerPasscodeDisabled(17),
+    unexpectedCommissionerPasscodeReady(18);
```

Comment:
Add another data member to this enum (say "String description") and set the string values for each of these enum values.

Reply:
Ok, this will require a private and public constructor for this enum.